config/config.php

```diff
@@ -88,12 +88,7 @@
 
     'proxy' => null, // TODO inherit from moodle conf see http://moodle.local/admin/settings.php?section=http for more.
 
-    'authproc.sp' => array(
-        50 => array(
-            'class' => 'core:AttributeMap',
-            'oid2name',
-        ),
-    ),
+    'authproc.sp' => auth_plugin_saml2::saml2_authproc_filters_hook(),
```

Comment:
look good, but please don't remove the core:AttributeMap one that was already there. Or move this into saml2_authproc_filters_hook

Reply:
moved this to saml2_authproc_filters_hook as a default hook